pyproject.toml

```diff
@@ -1,3 +1,98 @@
+[project]
+name = "ops"
+description = "The Python library behind great charms"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    { name="The Charm Tech team at Canonical Ltd.", email="charm-tech@lists.launchpad.com" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+]
+dependencies = [
+    "PyYAML==6.*",
+    "websocket-client==1.*",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+docs = [
+    "sphinx==6.2.1",
+    "sphinx-design",
+    "furo",
+    "sphinx-tabs",
+    "lxd-sphinx-extensions",
+    "sphinx-copybutton",
+]
+dev = [
+    "isort~=5.11",
+    "autopep8~=1.6",
+    "flake8~=6.1",
+    "flake8-docstrings~=1.7",
+    "flake8-builtins~=2.1",
+    "pyproject-flake8~=6.1",
+    "pep8-naming~=0.13",
+    "pytest~=7.2",
+    "pyright==1.1.317",
+    "pytest-operator~=0.23",
+    "coverage[toml]~=7.0",
+    "setuptools>=60",
+    "setuptools-scm>=8.0",
+    "typing_extensions~=4.2",
+]
+
+[project.urls]
+"Homepage" = "https://juju.is/docs/sdk"
+"Repository" = "https://github.com/canonical/operator"
+"Issues" = "https://github.com/canonical/operator/issues"
+"Documentation" = "https://ops.readthedocs.org"
```

Comment:
ops.readthedocs.io seems to be the canonical URL, is that right?

Reply:
Ah, indeed (although it probably should not be). Thanks!